### ISSUE TITLE
Crosswalk And Document eoAPI Tests

### ISSUE THREAD

**Issue by ranchodeluxe** [developmentseed/eoapi-k8s#88]
# Context
Runtime tests got [moved over to this repo and expanded upon](https://github.com/developmentseed/eoAPI-devseed) it seems. Originally `eoapi-k8s` copy pasta'd them to make sure things worked across cloud providers. Now given #86 there's a potential third repo that tests might need to get

# Goal
Document which tests we have for `eoapi-devseed` and `eoapi-k8s`. Make decisions about which ones `eoepca+` needs. Then come up with a plan to "reuse" just the tests as much as possible between the projects where they need to be reused

# AC
- [ ] google doc matrix of tests between `eoapi-devseed` and `eoapi-k8s`
- [ ] ask around about ideas for sharing
- [ ] decide how to share and expand next ACs

**Comment by vincentsarago** [comment]
the tests were moved to eoapi-devseed because they were testing the custom runtimes, not the classic eoAPI building blocks. 

we could design tests for the raster/stac/vector API, but those should and are already well tested individually on their own repo.

**Comment by ranchodeluxe** [comment]
> we could design tests for the raster/stac/vector API, but those should and are already well tested individually on their own repo.

yeah, we don't need all the tests just s subset of them working for `eoapi-k8s` and then reused on `eoepca+` is how I see this now. Copy pasted for now between eoapi-k8s and eoepcia+. Will break out into a package later. 

But I believe these are needed b/c they've been pretty useful to have for testing out SSL, traefik and nginx rewrite capabilities to know when the proxy settings are misaligned 👍

**Comment by j08lue** [comment]
Orphaned
